ts/main.ts

```diff
@@ -425,3 +448,14 @@ $exerciseDetailSection.addEventListener('click', (event: Event) => {
 $favoritesCta.addEventListener('click', () => {
   viewSwap('exercises-view');
 });
+
+$newWorkoutForm.addEventListener('submit', (event: Event) => {
+  event.preventDefault();
+  const $formElements = $newWorkoutForm.elements as FormElements;
+  const newWorkout: Workout = {
+    name: $formElements.name.value,
+    days: parseInt($formElements.days.value),
+    exercises: [],
+  };
+  fitlogData.workouts.push(newWorkout);
+});
```

Comment:
Should this redirect the user after making the workout? Like go to the Exercises pages so they can search for workouts to add?